ComfyUI/custom_nodes/luma_photon_node/luma_photon_node.py

```
@@ -0,0 +1,250 @@
+            result_url_extractor=image_result_url_extractor,
+            node_id=unique_id,
+            auth_kwargs=auth_kwargs,
+        )
+        response_poll = operation.execute()
+
+        try:
+            img_response = requests.get(response_poll.assets.image)
+            img_response.raise_for_status()
+            img = process_image_response(img_response)
+            return (img, depth_map_to_return)
+        except requests.RequestException as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)